src/components/CosmeticShop/cosmetic-shop.util.ts

```diff
@@ -114,6 +115,15 @@ export const useMutateCosmeticShop = () => {
     }
   };
 
+  const upsertCosmeticMutation = trpc.cosmeticShop.upsertCosmetic.useMutation({
+    async onSuccess() {
+      await queryUtils.cosmeticShop.getShopItemsPaged.invalidate();
+    },
+    onError(error) {
+      onError(error, 'Failed to update or create the cosmetic');
+    },
+  });
+
```

Comment:
Seems like we are not using this anymore, we can keep it if you think it makes sense